src/com/google/common/css/compiler/gssfunctions/GssFunctions.java

```diff
@@ -370,7 +375,7 @@ public String getCallResultString(List<String> args)
           baseColorString, args.get(1), args.get(2), args.get(3));
     }
 
-    protected String addHsbToCssColor(
+    public static String addHsbToCssColor(
```

Comment:
Why the change to `public static`? The idea with `protected` is to indicate that it's *not* part of the public interface of the class for callers, but can be used by subclasses. I know that there is the `parseBoolean` method above, but I don't think it should be `public static` either. What do you think?